### PR TITLE
spelling fixes on docs

### PR DESCRIPTION
**Pull request by h00die**
Fix spelling in docs.

I tried to go 'easy' mode, not change things like `cant` to `can't` . Also attempted to leave spelling issues that look like they came from other sources (like Synology's output has a spelling error that the module outputs).

Yes, some of the spelling issues come from module output, modules are next.

### CODE REVIEW

File: documentation/modules/auxiliary/admin/kerberos/keytab.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ Export Kerberos encryption keys stored in the Metasploit database to a keytab fi
 # Secrets dump
 msf6 > use auxiliary/gather/windows_secrets_dump
 msf6 auxiliary(gather/windows_secrets_dump) > run smbuser=Administrator smbpass=p4$$w0rd rhosts=192.168.123.13
-... ommitted ...
+... omitted ...
```
Comment (adfoster-r7) [inline documentation/modules/auxiliary/admin/kerberos/keytab.md:66]: This was me, the shame

File: documentation/modules/auxiliary/scanner/http/http_put.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ This module can abuse misconfigured web servers to upload and delete web content
 2. Do: ```set RHOSTS [IP]```
 3. Do: ```set RPORT [PORT]```
 4. Do: ```set PATH [PATH]```
-5. Do: ```set FILENAME [FILNAME]```
+5. Do: ```set FILENAME [FILENAME]```
```
Comment (adfoster-r7) [inline documentation/modules/auxiliary/scanner/http/http_put.md:10]: I guess this is technically broken docs now, but the `Did you mean $x?` functionality should kick in if the user typos anyway

File: documentation/modules/exploit/linux/samba/is_known_pipename.md [hunk 3]
```diff
@@ -18,7 +18,7 @@ where no SMB options are required to be set:
 comment = CVE-2017-7494
 path = /tmp
 writable = yes
-browseable = yes
```
Comment (adfoster-r7) [inline documentation/modules/exploit/linux/samba/is_known_pipename.md:21]: Needs reverted I think - https://www.samba.org/samba/docs/current/man-html/smb.conf.5.html#BROWSEABLE

File: documentation/modules/exploit/multi/http/magento_unserialize.md [hunk 4]
```diff
@@ -68,7 +68,7 @@ If at some point the IP (base URL) of Magento has changed, then you will need to
 
 1. From the terminal, do: ```mysql -h localhost -u [username] -p[password]```
 2. In the SQL prompt, do: ```use [magento database name]```
-3. Do: ```select * from core_config_data;```, you should see both web/unsecure/base_url (config ID 2) and web/secure/base_url (config ID 3) with the hardcoded IP.
+3. Do: ```select * from core_config_data;```, you should see both web/insecure/base_url (config ID 2) and web/secure/base_url (config ID 3) with the hardcoded IP.
```
Comment (adfoster-r7) [inline documentation/modules/exploit/multi/http/magento_unserialize.md:71]: I think this needs reverted https://magento.stackexchange.com/questions/227157/why-is-web-unsecure-base-url-missing-in-core-config-data

**Review by adfoster-r7 — CHANGES_REQUESTED** [review]
Just one or two need reverted I think

File: documentation/modules/auxiliary/admin/kerberos/keytab.md (hunk 1 above)
Comment (h00die) [reply documentation/modules/auxiliary/admin/kerberos/keytab.md:66]: I fixed a few of my own as well. happens to everyone!

File: documentation/modules/auxiliary/scanner/http/http_put.md (hunk 2 above)
Comment (h00die) [reply documentation/modules/auxiliary/scanner/http/http_put.md:10]: I think you have the variable name and the identifier mixed up, this change has no change on the actual things being typed by the user

Comment (adfoster-r7) [reply documentation/modules/auxiliary/scanner/http/http_put.md:10]: 💯

**Comment by h00die** [comment]
2 reverts out of 309, not too bad!

**Comment by adfoster-r7** [comment]
## Release Notes

Fixes spelling mistakes in module documentation

**Pull request merged**